pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/JavaInstanceStarter.java

```diff
@@ -31,6 +31,8 @@
 import io.grpc.stub.StreamObserver;
 import io.prometheus.client.CollectorRegistry;
 import io.prometheus.client.exporter.HTTPServer;
+import io.prometheus.client.hotspot.*;
```

Comment:
This should probably not be a wildcard import, those are pretty heavily frowned upon in java land